README.md

```diff
@@ -48,3 +48,63 @@ Requests to cross-scheme subresources will no longer send SameSite=Lax or SameSi
 
 For example, if http://site.example embeds an image from https://site.example, SameSite=Lax and SameSite=Strict cookies will no longer be sent on the request to https://site.example, and any such cookies in the response will be ignored. 
 
+## Questions
+### How do Schemeful Same-Site and Scheme-Bound Cookies differ?
+
+"Schemeful Same-Site" and "[Scheme-Bound Cookies](https://github.com/mikewest/scheming-cookies)" are both trying to move cookies closer to an origin based security model; these two proposals complement one another and one is not a subset of the other.
+
+#### Schemeful Same-Site
+"Schemeful Same-Site", and the current version of same-site, is concerned with the browsing context in which the cookie will be sent or recieved. The browsing context can be thought of as the answer to the question "Is the request/response url the same site as the one I'm on?" More specifically "Is the request/response url 'same-site' or 'cross-site' with the current site I'm on?" 
```

Comment:
Add a sentence saying that "the site I'm on" is generally considered the site in the URL bar i.e. the top level site.